### PR TITLE
feat(snapshots): added ability to delete latest snapshot

### PR DESCRIPTION
**Pull request by petterannerwall**
On hold until backend has been resolved for this

### CODE REVIEW

File: ExilenceNextApp/src/components/toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -266,6 +268,23 @@ const Toolbar = ({
...
               </Tooltip>
+              <Tooltip title={t('label.remove_snapshot_icon_title') || ''} placement="bottom">
+                <span>
+                  <IconButton
+                    disabled={
+                      !activeProfile ||
+                      isSnapshotting ||
+                      !signalrOnline ||
+                      activeProfile.snapshots.length === 0
+                    }
```
Comment (sbsrnt) [inline ExilenceNextApp/src/components/toolbar/Toolbar.tsx:279]: I suggest moving logic outside of props and just passing `isDisabled` variable

File: ExilenceNextApp/src/components/toolbar/Toolbar.tsx [hunk 2]
```diff
@@ -266,6 +268,23 @@ const Toolbar = ({
...
+                    onClick={() => handleClearSnapshot()}
```
Comment (sbsrnt) [inline ExilenceNextApp/src/components/toolbar/Toolbar.tsx:280]: `onClick={() => handleClearSnapshot()}` can be shortened to `onClick={handleClearSnapshot}`

**Review by sbsrnt — APPROVED** [review]

File: ExilenceNextApp/src/components/toolbar/Toolbar.tsx (hunk 1 above)
Comment (viktorgullmark) [reply ExilenceNextApp/src/components/toolbar/Toolbar.tsx:279]: Agreed, but for now this is fine IMO as its consistent with how the logic looks right now in the entire toolbar component. I think we have some old issue lying around related to refactoring this monster though 👹

**Comment by viktorgullmark** [comment]
We should add a confirmation dialog similar to how its done for clearing all snapshots, see ToolbarContainer.tsx:137

**Pull request closed without merging**